__tests__/app.test.js

```diff
@@ -241,4 +241,52 @@ describe("GET /api/users", () => {
   });
 });
 
-
+describe("GET /api/reviews", () => {
+  test("200: returns an array", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeInstanceOf(Array);
+      });
+  });
+  test("200: returns an array of the correct length", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews.length).toBe(13);
+      });
+  });
+  test("200: reviews have the correct properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        reviews.forEach((review) => {
```

Comment:
make sure you are not getting an empty array back here!

Otherwise you could be getting false-postiives from your tests!